app/src/main/java/com/example/taskyy/data/mappers/UserToUserDTOMapper.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.taskyy.data.mappers
+
+import com.example.taskyy.data.remote.data_transfer_objects.RegisterUserDTO
+import com.example.taskyy.domain.objects.User
+import javax.inject.Inject
+
+class UserToUserDTOMapper @Inject constructor(
+) {
+
+    fun mapUserToUserDTO(user: User): RegisterUserDTO {
+        return RegisterUserDTO(fullName = user.fullName, email = user.email, password = user.password)
+    }
+}
```

Comment:
I personally never found use in making mappers a separate class. I love just making them normal extension functions: 
```
fun User.toRegisterUserDTO(): RegisterUserDTO {
    // ...
}
```